src/main/java/com/coyoapp/tinytask/dto/TaskRequest.java

```diff
@@ -14,5 +21,8 @@
 
   @NotEmpty
   private String name;
+  @JsonFormat(pattern = "dd MMM yyyy")
```

Comment:
Using this pattern, I get a backend error: "JSON parse error: Cannot deserialize value of type `java.util.Date` from String "25 Mar 2021": expected format "dd MMM yyyy"
Apparently moment does really have another formatting (especially when it comes to capitals), but it would have been safer to only go with numbers like dd-MM-yyyy anyway.

Reply:
I don't know but it is working fine on my system.